### ISSUE TITLE
The number of contributors shown in the top header on the generated webpage seems too big

### ISSUE THREAD

**Issue by mariusc** [47deg/org#55]
Is there any way to exclude specific repositories that are on the GitHub organization page from the website? I'm thinking about forks of other projects. 

I don't add them to `:included projects`, but it seems like their number of stars and contributors still counts for the statistics in the site header (the onse showing th enubmer of stars, contributors, repos and languages)

**Comment by mariusc** [comment]
Actually, I looked into this a bit more, and I think that was not the cause of the issue I was seeing. 

The problem I noticed was that the number of contributors shown in the top banner of the webpage seemed to be too high. I suspected this was because of a fork we had on our organization. But actually, after deleting the fork the number of contributors still seemed to be a bit too big. I'm thinking maybe the total number of contributors doesn't exclude duplicates (same contributors working on multiple repos in the organization). Could this be the case?
